### PR TITLE
Update: Added cart and wishlist page

### PR DESCRIPTION
**Pull request by HeyNitin**
image
image

### CODE REVIEW

File: Pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+              <div class="badge-square">
+                <i class="fa fa-heart"></i>
+              </div>
+              <footer class="footer">
+                <button class="button btn-primary">Move to Cart</button>
+                </footer>
+                </div>
+        
+    
+    </div>
```
Comment (Mohasin-Haque) [inline Pages/wishlist.html:198]: remove whitespace or run prettier once

**Review by Mohasin-Haque — COMMENTED** [review]
Your code looks good but you can remove spaces and also can give comments.

**Review by rohanmathur91 — APPROVED** [review]
Looks good to me

File: Pages/products.html [hunk 2]
```diff
@@ -196,14 +184,13 @@
...
-              favorite_border
-            </i>
+            <i class="fa fa-heart-o"></i>
           </div>
           <footer class="footer">
             <button class="button btn-primary">Add to Cart</button>
             </footer>
             </div>
+      
       </div>
```
Comment (RakshaPawar108) [inline Pages/products.html:195]: Remove unnecessary spaces by running the code through Prettier once.

File: Pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/82dc64715a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/Styles/wishlist.css" />
+    <title>Wishlist</title>
+  </head>
+
+  <body>
+    <nav class="nav-header">
```
Comment (RakshaPawar108) [inline Pages/wishlist.html:13]: Good use of semantic HTML!

File: Styles/header.css [hunk 4]
```diff
@@ -18,9 +18,12 @@
 }
 
 .nav-header .button {
-  height: fit-content;
+  height: 1.5rem;
+  width: 4rem;
+  text-align: center;
   background-color: wheat;
```
Comment (RakshaPawar108) [inline Styles/header.css:24]: You can give a variable to the `wheat` colour and use its hex code instead of directly the name

**Review by RakshaPawar108 — APPROVED** [review]
Apart from the minor suggested changes, the code looks good to merge! Great work!

**Pull request merged**